src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -149,31 +152,33 @@ public RowIdMatch mapValuesToRows(bio.terra.metadata.Snapshot snapshot,
         String datasetTableName = column.getTable().getName();
         String datasetBqDatasetName = prefixName(source.getDataset().getName());
 
-        StringBuilder builder = new StringBuilder();
-        builder.append("SELECT T.")
-                .append(PDAO_ROW_ID_COLUMN)
-                .append(", V.inputValue FROM (SELECT inputValue FROM UNNEST([");
-
         // Put all of the values into an array that is unnested into a table
+        StringBuilder values = new StringBuilder();
         String prefix = "";
         for (String inval : inputValues) {
-            builder.append(prefix).append("'").append(inval).append("'");
+            values.append(prefix).append("'").append(inval).append("'");
             prefix = ",";
         }
 
-        builder.append("]) AS inputValue) AS V LEFT JOIN `")
-                .append(projectId)
-                .append('.')
-                .append(datasetBqDatasetName)
-                .append('.')
-                .append(datasetTableName)
-                .append("` AS T ON V.inputValue = T.")
-                .append(datasetColumnName);
+        String sqlTemplate =
+            "SELECT T.{pdaoRowIdColumn}, V.inputValue " +
```

Comment:
The DAO classes use a convention of `:templateField` instead of `{templateField}`, if we want to be consistent.

Reply:
The `StringSubstitutor` requires both a prefix and a suffix